xbmc/video/dialogs/GUIDialogCMSSettings.cpp

```diff
@@ -20,7 +20,7 @@
 
 #include "system.h"
 
-#ifdef HAS_GL
+#if defined(HAS_GL) || defined(HAS_DX)
```

Comment:
the define was here before cmake. now things can be improved when they are touched anyway.